Script/install.sh

```diff
@@ -11,115 +10,30 @@ REPO_PATH=$(readlink -f $(dirname $0) | grep -E -o "^(.*?\\GrovePi)")
 
 source $DEXTERSCRIPT/functions_library.sh
 
-identify_cie() {
```

Comment:
Why are those removed? They are needed for branding.

Reply:
There are put back in `update_grovepi.sh`. They are supposed to be there - otherwise you'll get a message with "Welcome to GrovePi's installer" somewhere in the middle on the installation or none at all.